cmd/katapult/data_centers_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/krystal/go-katapult"
+	"github.com/krystal/go-katapult/core"
+	"github.com/krystal/katapult-cli/cmd/katapult/mocks"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var dcs = []*core.DataCenter{
+	{
+		ID:        "POG1",
+		Name:      "hello",
+		Permalink: "Hello World!",
+		Country: &core.Country{
+			ID:   "POG",
+			Name: "Pogland",
+		},
+	},
+	{
+		ID:        "GB1",
+		Name:      "hello",
+		Permalink: "Hello World!",
+		Country: &core.Country{
+			ID:   "UK",
+			Name: "United Kingdom",
+		},
+	},
+}
+
+const stdoutDcList = ` - hello (Hello World!) [POG1] / Pogland
+ - hello (Hello World!) [GB1] / United Kingdom
+`
+
+func TestDataCenters_List(t *testing.T) {
```

Comment:
I think this should probably a be a tabular test, as there's at least three different scenarios that might happen:

- Successful response with list of Data Centers.
- Successful response but list of Data Centers is empty (should never happen in reality, but probably worth a quick test case).
- Error response received from the API for some reason.